src/main/app/src/app/shared/progress-snackbar/progress-snackbar.component.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2024 Lifely
```

new file?

Reply:
Same here; This is a new component but the code was ATOS and moved out of util.service.ts; so trivial I guess